classes/announcements-classes.py

```diff
@@ -0,0 +1,18 @@
+from datetime import datetime
+
+class announcement:
+  def __init__ (self, id, title, text)
```

Comment:
put each input parameter on a different line
add default values for any that should have them
don't import the `id` - we'll generate that upon class initialization

Reply:
ACTUALLY
1) Replace `id` with `uuid`
2) Generate the `uuid` inside the function, not as a parameter
https://docs.python.org/3/library/uuid.html

Reply:
Should I do the uuid for the group class as well? And can you explain how to implement the uuid?

Reply:
Yes, replace all `id`s with `uuid`, which will all be generated inside the init class

https://www.uuidgenerator.net/dev-corner/python